src/components/hand-controls.js

```diff
@@ -24,18 +24,9 @@ module.exports.Component = registerComponent('hand-controls', {
     this.onTrackpadUp = function () { self.handleButton('trackpad', 'up'); };
     this.onTriggerDown = function () { self.handleButton('trigger', 'down'); };
     this.onTriggerUp = function () { self.handleButton('trigger', 'up'); };
-    this.addEventListeners();
```

Comment:
I like to keep event listener attachment logic in a separate method. Can we keep that code as it was?